src/install/kafka/linux/install-linux-boot.mdx

```diff
@@ -0,0 +1,54 @@
+---
+componentType: default
+headingText: Configure the Kafka integration
+---
+
+1. Change directory to the integrations configuration folder by running:
+
+```shell
+cd /etc/newrelic-infra/integrations.d
+```
+
+2. Copy the sample configuration file by running:
+
+```shell
+sudo cp nri-kafka-config.yml.sample kafka-config.yml
+```
+
+3. Edit the `kafka-config.yml` file. The following example collects metrics and inventory discovered by the bootstrap broker. 
```

Comment:
I think you should expand on this first part a little bit more. Edit the file with the below changes, edit the file to enable XYZ, something like that.